repos/system_upgrade/el7toel8/actors/scanmemory/libraries/library.py

```diff
@@ -0,0 +1,22 @@
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+def _get_memoryinfo():
+    """Return lines from /proc/meminfo."""
```

Comment:
Please specify return data type in the docstring. Now you have either a dict or None, shouldn't you unify that?